sklearn/compose/_column_transformer.py

```diff
@@ -356,6 +357,53 @@ def _fit_transform(self, X, y, func, fitted=False):
             else:
                 raise
 
+    def _calculate_inverse_indices(self, X):
+        """
+        Private function to calcuate indicies for inverse_transform
+        """
+        # checks for overlap
+        all_indexes = set()
+        input_indices = []
+        for name, trans, cols in self.transformers:
+            col_indices = _get_column_indices(X, cols)
+            if not all_indexes.isdisjoint(set(col_indices)):
+                self._invertible = (False,
```

Comment:
This doesn't appear to be covered by tests.

Reply:
`test_column_transformer_inverse_transform_with_overlaping_slices` should cover this, I added the ValueError message in the assertion.